AtReconstruction/AtMacroTask.h

```diff
@@ -9,20 +9,29 @@
 class TClass;
```

Comment:
```
/user/anthonya/attpcroot/AtReconstruction/AtMacroTask.h should add these lines:
#include <Rtypes.h>    // for THashConsistencyHolder, ClassDef, Option_t
#include <algorithm>   // for max
class TBuffer;
class TMemberInspector;
```

Reply:
Why?

Reply:
if Rtypes is required for ClassDef, I can see that. No idea why the rest would be relevant.

Reply:
TBuffer and TMemberInspector are both used in the classdef and need to be forward declared to make IWYU happy so we are not spammed by every single TObject class. (Blame ROOT)